JapaneseFilter/sections/specific.txt

```diff
@@ -2584,6 +2584,7 @@ magnet.so###p_ad
 magnet.so###AD
 ceron.jp##div[style="width:320px;min-height:100px;margin: 0 auto;background:#eee;"]
 itmedia.co.jp###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]
+gendai.ismedia.jp##.blockContainer > aside[style="min-height:250px;"]
```

Comment:
I only see it in the mobile version of the website. Please move it to the `Mobile filter - specific_web`.